ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -113,6 +112,25 @@ enum InstitutionMode {
   EDIT,
 }
 
+interface CommonToggleProps {
+  name: string;
+  checked: boolean;
+  dataTestId: string;
+  onToggle: (boolean) => void;
+  disabled?: boolean;
+}
+const CommonToggle = (props: CommonToggleProps) => {
```

Comment:
since this is the same as the CommonToggle in admin-user-profile, we should extract it to a common location, maybe components/admin/common-toggle ?

Reply:
Good idea. I pulled it out.